### ISSUE TITLE
face from cells (warning)

### ISSUE THREAD

**Issue by theodore-michel** [DonsetPG/graph-physics#43]
Getting this warning at each training in logger: 
```
graphphysics/utils/torch_graph.py:175: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor). 
    face = torch.tensor(cells)
```

Which makes me wonder why @bazziamir24 has written it this way, and if we could just do `cells.clone().detach()` (since `cells` is already a tensor created in lines above) or create `face` another way:

https://github.com/DonsetPG/graph-physics/blob/715f377a1ff07533d2b5a473bda3faeb12ed80f7/graphphysics/utils/torch_graph.py#L174-L175

**Comment by DonsetPG** [comment]
or just `face = cells` unless i'm missing something

**Comment by theodore-michel** [comment]
closed by #47
